server.go

```diff
@@ -169,9 +232,58 @@ func (server *Server) pingRoute(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(response)
 }
 
+func (server *Server) versionRoute(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	response := VersionResponse{Version: WAGGLE_VERSION}
+	json.NewEncoder(w).Encode(response)
+}
+
+func (server *Server) signersHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodGet:
+		server.signersRoute(w, r)
+		return
+	case http.MethodPost:
+		var signer string
+		routePathSlice := strings.Split(r.URL.Path, "/")
+		requestedSigner := common.HexToAddress(routePathSlice[2])
+		for s := range server.AvailableSigners {
+			if s == requestedSigner.String() {
+				signer = s
+				break
+			}
+		}
+		if signer == "" {
+			http.Error(w, fmt.Sprintf("Unacceptable signer provided %s", signer), http.StatusBadRequest)
+			return
+		}
+
+		server.signDropperRoute(w, r, signer)
```

Comment:
Does `net/http` have something like FastAPI applications? So you could define a sub-mux or something?